### ISSUE TITLE
Insert SCHEDULED items too?

### ISSUE THREAD

**Issue by unhammer** [unhammer/org-mru-clock#4]
If an item is SCHEDULED for today, but not yet clocked in to, it'd be nice if it were there in the `org-mru-clock-in` list, as a "suggestion".

- Should we mark it in some way?
- How far up the list should it be, if it's never been clocked in to?
- Should we include things that were scheduled in the past, but not clocked in to? How far back then?
- Will this make us much slower?

**Comment by unhammer** [comment]
related: https://github.com/unhammer/org-mru-clock/tree/4/scheduled-items

**Comment by shrysr** [comment]
I agree that feature would be nice. I think it definitely should be marked in some way. Maybe all such scheduled tasks could be prefixed with some kind of keyword that makes them quickly accessible via ivy? Perhaps writing out a temporary cache file could address the speed. I would be happy to test (my elisp is not yet good enough to actually help you, I'm afraid)
